sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/MetricsQueryClient.java

```diff
@@ -144,7 +143,7 @@ public Response<MetricsQueryResult> queryResourceWithResponse(String resourceUri
         }
         String timespan = options == null || options.getTimeInterval() == null
             ? null
-            : LogsQueryHelper.toIso8601Format(options.getTimeInterval());
+            : MetricsHelper.toMetricsTimespan(options.getTimeInterval());
```

Comment:
Note: this is different behavior from what we have in MetricsClient where we [throw an exception](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/MetricsClient.java#L110-L113) if `duration` is set. If we want to support `duration`, then the MetricsClient should also be updated.